migrations/2022-11-14-194539_create_attributes/up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE nft_attributes (
+  nft_metadata_id  TEXT NOT NULL,
+  nft_contract_id TEXT NOT NULL,
+  attribute_type TEXT NOT NULL,
+  metadata_value TEXT,
```

Comment:
This should be `attribute_value` in accordance with `attribute_type`

Reply:
This should also be a `jsonb` typed field

Reply:
Change didn't get commited for some reason, resolved